packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -302,14 +303,17 @@ export class WalletConnectConnection implements WalletConnection {
         challenge: string,
         credentialStatements: CredentialStatements
     ): Promise<VerifiablePresentation> {
-        return this.connector.client.request<VerifiablePresentation>({
+        const serializedParams = JSONBigInt.stringify({ challenge, credentialStatements });
```

Comment:
Can't you use `jsonUnwrapStringify` from web-sdk? That's what we currently use in `accountTransactionPayloadToJson`.

Reply:
Also, wouldn't it make sense to only stringify `credentialStatements` as `challenge` is already a string?

Reply:
I have used `jsonUnwrapStringify` as it basically does the same in our case. Whether or not to include the `challenge` in what is stringified is mostly a stylistic choice from my perspective. I would rather just keep it as is, as on the Android side we have a generic wrapper for taking parameters as a JSON string.